setup.py

```diff
@@ -480,7 +480,7 @@ def get_file_list (self):
     },
     # Requirements, usable with setuptools or the new Python packaging module.
     install_requires = [
-        'requests >= 2.2.0',
+        'requests<2.15,>=2.2',
```

Comment:
@anarcat  2.2 is ok but  newer one (mb 2.15) is problematic. See linkchecker log https://travis-ci.org/syl20bnr/develop.spacemacs.org/builds/245720094?utm_source=github_status&utm_medium=notification